src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -42,7 +42,20 @@ object WorkflowActor {
 
   val DatabaseTimeout = 5 seconds
 
-  type ExecutionStore = Map[Call, ExecutionStatus.Value]
+  type ExecutionStore = Map[ExecutionStoreKey, ExecutionStatus]
+  type ExecutionStoreEntry = (ExecutionStoreKey, ExecutionStatus)
+
```

Comment:
Is the comment below about only being used for testing accurate?  I can see that it definitely is the case for this PR, but this is also the code we'll use to initially populate the table, right?